FootlessParser.xcodeproj/FootlessParser_Info.plist

```diff
@@ -18,7 +18,7 @@
   <key>CFBundleSignature</key>
   <string>????</string>
   <key>CFBundleVersion</key>
-  <string>$(CURRENT_PROJECT_VERSION)</string>
+  <string>186</string>
```

Comment:
Where does 186 come from?

Reply:
Apologies, this was completely my mistake!  This was done for some internal testing and was suppose to go against a custom fork!